sbin/build.sh

```diff
@@ -647,30 +657,59 @@ removingUnnecessaryFiles() {
   # but if they were explicitly requested via the configure option
   # '--with-native-debug-symbols=(external|zipped)' leave them alone.
   if [[ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_OPENJ9}" ]] ; then
-    # .diz files may be present on any platform
-    # Note that on AIX, find does not support the '-delete' option.
-    find "${jdkTargetPath}" "${jreTargetPath}" -type f -name "*.diz" | xargs rm -f || true
+    deleteDebugSymbols
+  fi
 
+  if [ ${BUILD_CONFIG[CREATE_DEBUG_SYMBOLS_PACKAGE]} == true ]; then
     case "${BUILD_CONFIG[OS_KERNEL_NAME]}" in
       *cygwin*)
-        # on Windows, we want to remove .map and .pdb files
-        find "${jdkTargetPath}" "${jreTargetPath}" -type f -name "*.map" -delete || true
-        find "${jdkTargetPath}" "${jreTargetPath}" -type f -name "*.pdb" -delete || true
+        # on Windows, we want to take .pdb files
+        debugSymbols=$(find "${jdkTargetPath}" -type f -name "*.pdb")
         ;;
       darwin)
-        # on MacOSX, we want to remove .dSYM folders
-        find "${jdkTargetPath}" "${jreTargetPath}" -type d -name "*.dSYM" | xargs -I "{}" rm -rf "{}"
+        # on MacOSX, we want to take .dSYM folders
+        debugSymbols=$(find "${jdkTargetPath}" -print -type d -name "*.dSYM")
```

Comment:
Is the `-print` necessary?

Reply:
Yes, to get the actual files inside the `.dSYM` folders and then be copied by `cpio`. If not, only folders are copied.